eng/Versions.props

```diff
@@ -118,6 +118,7 @@
     <SystemRuntimeCompilerServicesUnsafeVersion>6.0.0-preview.2.21101.2</SystemRuntimeCompilerServicesUnsafeVersion>
     <!-- System.Security.AccessControl should only be referenced in Dependencies.props and RTMVersions.csproj. -->
     <SystemSecurityAccessControlVersion>6.0.0-preview.2.21101.2</SystemSecurityAccessControlVersion>
+    <SystemSecurityCryptographyAlgorithmsVersion>6.0.0-preview.2.21101.2</SystemSecurityCryptographyAlgorithmsVersion>
```

Comment:
The package doesn't appear to be used. Do you know why this is required?

Reply:
Nope I was just doing this out of habit, I'll remove then

Reply:
I think this is needed by Dependenices.props since I got a code check failure when i removed this

Reply:
Do you know what the error was? Weird that a version is required here without it being used.

Reply:
https://dev.azure.com/dnceng/public/_build/results?buildId=974536&view=logs&j=efda9e5f-cf98-536e-4181-9b6c13ac35b3&t=de5280bb-c511-517b-1959-798dcf9a84da&l=740

##[error]Missing version variable 'SystemSecurityCryptographyAlgorithmsVersion' in the 'Automated' property group in F:\workspace\_work\1\s/eng/Versions.props

Reply:
Checking that Versions.props and Version.Details.xml match

So I'm guessing I would need to remove the one in version.details.xml too?

Reply:
yes please. Let's remove it from both locations.